NetworkLayer/Model/Category.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Category.swift
+//  NetworkLayer
+//
+//  Created by Pierre on 15/11/2022.
+//
+
+import Foundation
+
+struct Category: Codable {
```

Comment:
```suggestion
struct Category: Decodable {
```

Utilise "Decodable" tu cherches simplement à décoder.